app/questionnaire/placeholder_transforms.py

```diff
@@ -32,12 +32,14 @@ def __init__(
         language: str,
         schema: QuestionnaireSchema,
         renderer: "PlaceholderRenderer",
+        block_on_routing_path: bool = False,
     ):
         self.language = language
         self.schema = schema
         self.renderer = renderer
         self.locale = DEFAULT_LOCALE if language in {"en", "eo"} else language
         self.ops_helper = OperationHelper(self.locale, self.schema, self.renderer)
+        self.block_on_routing_path = block_on_routing_path
```

Comment:
Could we rename this var in a boolean way too?

Reply:
updated to `is_block_on_routing_path`